roles/ohpc_slurm_17_11_11/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+  - name: Extract Slurm 17.11.11 RPMs
+    unarchive:
+      src: "{{ slurm_17_11_11_url }}"
+      dest: "{{ slurm_17_11_11_dest }}"
+      remote_src: yes
+
+  - name: Update slurm related packages on ohpc node
+    yum:
+      name:
+        - "{{ slurm_17_11_11_dest }}/slurm17.11.11/slurm-devel-ohpc-17.11.11-6.2.ohpc.1.3.6.x86_64.rpm"
+        - "{{ slurm_17_11_11_dest }}/slurm17.11.11/slurm-example-configs-ohpc-17.11.11-6.2.ohpc.1.3.6.x86_64.rpm"
+        - "{{ slurm_17_11_11_dest }}/slurm17.11.11/slurm-ohpc-17.11.11-6.2.ohpc.1.3.6.x86_64.rpm"
+        - "{{ slurm_17_11_11_dest }}/slurm17.11.11/slurm-perlapi-ohpc-17.11.11-6.2.ohpc.1.3.6.x86_64.rpm"
+        - "{{ slurm_17_11_11_dest }}/slurm17.11.11/slurm-slurmctld-ohpc-17.11.11-6.2.ohpc.1.3.6.x86_64.rpm"
+        - "{{ slurm_17_11_11_dest }}/slurm17.11.11/slurm-slurmdbd-ohpc-17.11.11-6.2.ohpc.1.3.6.x86_64.rpm"
+
+  - name: Update slurm related packages on compute node
```

Comment:
Change the  text to read "...for compute node image" .  We aren't changing this directly on a running compute node but rather maintaining the image that boots the compute node.